db/flush_job.cc

```diff
@@ -325,16 +326,18 @@ Status FlushJob::WriteLevel0Table() {
       }
       total_num_entries += m->num_entries();
       total_num_deletes += m->num_deletes();
+      total_data_size += m->data_size();
       total_memory_usage += m->ApproximateMemoryUsage();
     }
 
-    event_logger_->Log()
-        << "job" << job_context_->job_id << "event"
-        << "flush_started"
-        << "num_memtables" << mems_.size() << "num_entries" << total_num_entries
-        << "num_deletes" << total_num_deletes << "memory_usage"
-        << total_memory_usage << "flush_reason"
-        << GetFlushReasonString(cfd_->GetFlushReason());
+    event_logger_->Log() << "job" << job_context_->job_id << "event"
+                         << "flush_started"
+                         << "num_memtables" << mems_.size() << "num_entries"
```

Comment:
Is this the result of `make format`? I'm thinking maybe the following is easier to read.
```
...
<< "num_memtables" << mems_.size()
<< "num_entries" << total_num_entries
<< "num_deletes" << total_num_deletes
...
```
Basically keep the name of the variable with its actual value in the same line.

Reply:
Yes, this is done by internal lint...

Reply:
LGTM